### DISCUSSION TITLE
Can I delete plot files in farming status ?

### DISCUSSION THREAD

**Discussion by DylannJ** [Chia-Network/chia-blockchain#2988] [Q&A]
I need more space for farming and can't buy ssd/hdd, so if I delete some file, does it have an impact ?

I test something, delete 6 k25 on my final directory, and in chia software they are not deleted tthey are in farming status, does that mean if they status change for farming we can delete them ?

**Comment by Keriwaza** [comment]
Not sure if k25 will work with the mainnet. I thought only k32. You farm with your plots, delete a plot and you've lost part of your farm.

**Reply by DylannJ** [reply]
k25 not work actually but I save him maybe in futur the k25 will work, ok I see

**Comment by cyr616** [comment]
k25 will never work in the future，it just for test. k32 will not work in a few years，you will need k33 and so on.
